regexp.c

```diff
@@ -54,30 +45,22 @@ cRegexp::~cRegexp(void)
 void cRegexp::Compile()
 {
   FreeCompiled();
-  const char *error;
-  int erroffset;
-  re = pcre_compile(regexp, modifiers, &error, &erroffset, NULL);
-  if (error) {
-     error("PCRE compile error: %s at offset %i", error, erroffset);
+  PCRE2_UCHAR error[256];
+  PCRE2_SIZE erroffset;
+  int errnumber;
+  re = pcre2_compile((PCRE2_SPTR8)regexp, PCRE2_ZERO_TERMINATED, modifiers, &errnumber, &erroffset, NULL);
+  if (errnumber) {
```

Comment:
This for some reason needs to be "if (!re)" for the code to work. Without the change I got "ERROR: PCRE compile error: no error at offset 0"

Reply:
Guess a lack of my understanding / reading on errn7mber. But !re/cre achieves the same. 👍